packages/web/src/scss/components/Grid/README.md

```diff
@@ -147,3 +147,125 @@ on larger breakpoints, you can combine Grid Span with responsive Grid columns:
 
 👉 Keep in mind Grid Span modifiers are designed to work only with a 12-column
 Grid (`.Grid[--cols-12]`, `.Grid--tablet--cols-12`, `.Grid--desktop--cols-12`).
+
+## Grid Item
+
+Grid Item is a wrapper for Grid items. It allows you to configure your Grid structure
+as you wish. Use CSS variables to set a column where the Grid Item should start, end or span over.
+If you set both end and span, end will take precedence.
+
+These variables are available:
+
+- `--grid-item-column-start`
+- `--grid-item-column-end`
+- `--grid-item-column-span`
```

Comment:
When I played around with it:

- setting `--grid-item-column-end` alone works,
- setting `--grid-item-column-span` alone works,
- setting `--grid-item-column-start` alone does **not** work.

Questions:

1. Does it mean that `span` or `end` is required for `start` to work? (Well, it does, that's what I found out.)
2. Shall we present it in the README?
3. Or, are we able to make `GridItem` work with any number of "arguments" (i.e. custom properties)?

Reply:
1. I split grid-column shorthand to its parts, so now each item is option. And set fallback to initial
2. not needed
3. yes, I only need to document how to set spans (eg. columnStart="1" or columnStart="span 2")